src/DynamoPackages/PackageLoader.cs

```diff
@@ -94,12 +100,8 @@ public string DefaultPackagesDirectory
         /// <param name="userDataFolder"></param>
         /// <param name="extension">subdirectory or subpath</param>
         /// <returns>combined root and extension path</returns>
-        private static string TransformPath(string root, string userDataFolder, string extension)
+        private static string TransformPath(string root, string extension)
```

Comment:
can you update the summary - I have a hard time understanding what this function does - and hows its changed.

Reply:
Yes, I'll update it. This is what it does: 
```
Combines the extension with the root path and returns it if the path exists. 
         If not, the root path is returned unchanged.
```